src/uwtools/resources/workflow.jsonschema

```diff
@@ -1,92 +1,101 @@
 {
```

Comment:
Let's not do anything with this file at all for now since it's out of scope for the changes to the Forecast validation component. Instead, we'll keep a pretty clean history that it came in as part of the first validation tool. We should circle back around and clean it up with what we require from a workflow-level validation schema.

I am seeing that the changes are mostly whitespace and a couple of list formatting changes. I'm not super familiar with JSON formatting, but a quick search indicates that the 4 space indentation is standard. Does it make sense for us to also keep that standard?

Reply:
There is a variety of opinion about 2 vs 4. We can either accept the `jq` default of 2, or use `--indent 4` and standardize on 4. I don't have an opinion, except that we should use automated formatting so that everyone follows the same rules and doesn't have to do manual formatting. The one fact I can offer is that `--indent 4` makes for a longer command. @christinaholtNOAA if you'd like to make an executive decision we should go with that.

Reply:
@WeirAE In case we don't hear back on this, it might be worth re-doing the `jq` formatting with `--indent 4`. If we decide to stick with 4-space indents for JSON, I'll make sure it gets in the Contributor Guide at some point.